### PR TITLE
Verify category

### PR DESCRIPTION
**Pull request by Pentapatch**

### CODE REVIEW

File: ShopAdmin/Commands/Category.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        }
+
+        public void Checkempty()
+        {
+            _logger.LogInformation("CheckEmpty starting");
+
+            string directoryPath = "outfiles\\category\\";
+            string filePath = $"{directoryPath}missingproducts-{DateTime.UtcNow:yyyyMMdd}.txt";
+
+            var categories = _categoryService.GetTrendingCategories(100); // TODO: Custom implementation
```
Comment (AronAsk) [inline ShopAdmin/Commands/Category.cs:26]: Remove comments :3

File: ShopAdmin/Commands/Category.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            var products = _productService.GetAllProducts(); // TODO: Custom implementation
+
+            // Abstract away to an interface
```
Comment (AronAsk) [inline ShopAdmin/Commands/Category.cs:29]: Remove Comments

File: ShopAdmin/Commands/Category.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                        Console.WriteLine($"Category {category.Id} {category.Name} has at least one product linked.");
+                        isFound = true;
+                        break;
+                    }
+                }
+
+                if(!isFound)
+                {
+                    Console.WriteLine($"Category {category.Id} {category.Name} has at least one product linked.");
+                    // TODO: Add kategorins namn to the list of results
```
Comment (AronAsk) [inline ShopAdmin/Commands/Category.cs:54]: Remove Comments

File: ShopAdmin/Commands/Category.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    {
+                        Console.WriteLine($"Category {category.Id} {category.Name} has at least one product linked.");
+                        isFound = true;
+                        break;
+                    }
+                }
+
+                if(!isFound)
+                {
+                    Console.WriteLine($"Category {category.Id} {category.Name} has at least one product linked.");
```
Comment (AronAsk) [inline ShopAdmin/Commands/Category.cs:53]: This line should be changed to "category does not have a product linked"

File: ShopAdmin/Commands/Category.cs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            var products = _productService.GetAllProducts(); // TODO: Custom implementation
+
+            // Abstract away to an interface
+            // Figure out tests:
+            //   - Kategorier som har kopplade produkter bör inte loggas
+            //   - Kategorier som inte har kopplade produkter bör loggas
+            //   - Rätt kategori loggas
+
+            // TODO: Declare the list here
+            List<string> missingProducts = new();
```
Comment (AronAsk) [inline ShopAdmin/Commands/Category.cs:36]: change name to "categoriesWithNoProducts" Or something else better

**Review by AronAsk — CHANGES_REQUESTED** [review]
Fix the comments, otherwise Well Done!

**Comment by NiclasEdstrom** [comment]
Fixed the comments

**Pull request merged**